test/functional/rpc_dumptxoutset.py

```diff
@@ -42,16 +39,38 @@ def run_test(self):
         with open(str(expected_path), 'rb') as f:
             digest = hashlib.sha256(f.read()).hexdigest()
             # UTXO snapshot hash should be deterministic based on mocked time.
-            assert_equal(
-                digest, '7ae82c986fa5445678d2a21453bb1c86d39e47af13da137640c2b1cf8093691c')
+            assert_equal(digest, expected_digest)
 
         assert_equal(
             out['txoutset_hash'], 'd4b614f476b99a6e569973bf1c0120d88b1a168076f8ce25691fb41dd1cef149')
         assert_equal(out['nchaintx'], 101)
 
         # Specifying a path to an existing file will fail.
```

Comment:
```suggestion
        self.log.info("Test that a path to an existing file will fail")
```

I would put some 'logs' on this test to follow each step when the test is running.

Reply:
Done. Suggestion accepted.